src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,80 @@
+package core.basesyntax;
+
+import core.basesyntax.models.*;
+
+public class Main {
+    public static void main(String[] agrs) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        Figure[] figures = new Figure[6];
+
+        for (int i = 0; i < 3; i++) {
+            figures[i] = figureSupplier.getRandomFigure();
+        }
+
+        for (int i = 3; i < 6; i++) {
+            figures[i] = figureSupplier.getDefaultFigure();
+        }
+
+        for (Figure figure : figures) {
+            System.out.println("Figure: " + getFigureName(figure)
+                    + ", area: " + figure.getArea()
+                    + " sq. units"
+                    + getFigureDescription(figure)
+                    + ", color: " + getColor(figure));
+        }
+    }
+
+    private static String getFigureName(Figure figure) {
+        if (figure instanceof Square) {
+            return "square";
+        } else if (figure instanceof Rectangle) {
+            return "rectangle";
+        } else if (figure instanceof RightTriangle) {
+            return "triangle";
+        } else if (figure instanceof Circle) {
+            return "circle";
+        } else if (figure instanceof IsoscelesTrapezoid) {
+            return "trapezoid";
+        } else {
+            return "";
+        }
+    }
+
+    private static String getFigureDescription(Figure figure) {

```

Comment:
Similarly to `getFigureName`, the `getFigureDescription` method should be refactored. Each figure class should have its own `getDescription` method that returns the specific description for that figure. This avoids the need for `instanceof` checks and makes the code more maintainable and scalable.